ApexDocContent/Opportunity.htm

```diff
@@ -54,7 +54,8 @@ <h2>Opportunity Classes</h2>
         <p> The classes involved with Opportunity Stage to State Mapping include:</p>
         <ul>
             <li>OPP_StageMappingUtil</li>
-            <li>OPP_StageMappingUtil_TEST</li>
+			<li>OPP_StageMappingUtil_TEST</li>
```

Comment:
The indentation looks off, but it could be my browser.  Can you confirm?

Reply:
I just pulled the code locally and the spacing looks fine. Not sure why it looks like this on the diff.